platforms/android/library/src/test/kotlin/io/element/android/wysiwyg/utils/HtmlToSpansParserTest.kt

```diff
@@ -118,15 +118,15 @@ class HtmlToSpansParserTest {
         """.trimIndent()
         val spanned = convertHtml(html)
 
-        assertThat(spanned.toString(), equalTo("Hey\nordered1\nordered2\nbullet1\nbullet2"))
+        assertThat(spanned.toString(), equalTo("Hey \nordered1\nordered2\nbullet1\nbullet2"))
```

Comment:
Maybe ` \n` (space and new line) could be replaced by `\n` (just new line)?

Reply:
You're probably right: I was checking the behaviour on an actual browser and I *thought* the whitespace was added, but checking again I think I got it wrong.